go/apps/http_api_nostream/resource.py

```diff
@@ -247,6 +247,13 @@ def handle_PUT_in_reply_to(self, request, payload, in_reply_to):
     @inlineCallbacks
     def handle_PUT_send_to(self, request, payload):
         user_account = request.getUser()
+        optout_store = self.worker.vumi_api.get_user_api(user_account) \
+            .optout_store
+        optout = yield optout_store.get_opt_out(
+            payload.get('to_addr_type', 'msisdn'), payload.get('to_addr'))
```

Comment:
I'm wondering if defaulting the to addr type to `msisdn` should happen in the message class to avoid us with different defaults all over the codebase. Don't think its something that should block this PR from landing though.